cmd/registry/cmd/config/set.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func setCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set PROPERTY VALUE",
```

Comment:
Do we expect users to want to set multiple properties with this command? This seems like it can be generalized to handle multiple properties without much difficulty.